frontend/src/app/home/home.page.html

```diff
@@ -1,28 +1,48 @@
 <ion-header [translucent]="true">
   <ion-toolbar>
     <ion-title>
-      Blank
+      Digitact
     </ion-title>
   </ion-toolbar>
 </ion-header>
 
 <ion-content [fullscreen]="true">
   <ion-header collapse="condense">
     <ion-toolbar>
-      <ion-title size="large">Blank</ion-title>
+      <ion-title size="large" style="color: grey;">Digitact</ion-title>
     </ion-toolbar>
   </ion-header>
 
   <div id="container">
-    <strong>Ready to create an app?</strong>
-    <p>
-      Start with Ionic
-      <a
-        target="_blank"
-        rel="noopener noreferrer"
-        href="https://ionicframework.com/docs/components"
-        >UI Components</a
-      >
+    <span style="color: red;">
+      Hello World!
+    </span>
+    <p style="color: black;">
+      Welcome to our new app
     </p>
+    <hr />
+    <form [formGroup]="checkOutForm">
+      <ion-item>
+        <ion-label>Name</ion-label>
+        <ion-input
+          required
+          type="text"
+          formControlName="firstName"
+          style="color: gray;"
```

Comment:
For future reference: You should apply inline styles as rarely as possible. Also, for Ionic components, in most cases you need to use CSS Variables (you can find them in the Ionic Docs, for instance https://ionicframework.com/docs/api/input#css-custom-properties ). I haven't tested it yet, but I believe that `style="color: gray;"` has no effect here.

This is no problem at the moment, since this is really just a "hello world" application.

Reply:
Okay, i will have a look on it, thanks :)